questionnaires/superset/README.md

```diff
@@ -112,3 +112,78 @@ docker-compose up
 ```
 docker-compose up
 ```
+
+## Share Dashboard
+
+In Superset, anyone can view a dashboard if they have the permanent link to it, whether or not they have an account on the Superset platform. But, they can only view the data in that dashboard if they have permission to read the dataset(s) for the dashboard.
+
+There are two things to consider when sharing a dashboard:
+- Do you want your dashboard (and dataset) to be available publicly? OR only to specific, authenticated users?
+- Do you want your dashboard to appear within the Superset interface, alongside the Menu buttons? OR do you want so share the dashboard without the typical Superset interface?
+
+The first step in sharing the dashboard is sharing the dataset - either publicly, or to specific authenticated users.
+
+The second step in sharing the dashboard is creating the permalink URL - showing the dashboard in the Superset interface, or on its own.
+
+### How to Share a Dataset Publicly
+
+This is a pre-requisite to publicly sharing your dashboard. Keep in mind that when you make this dataset public, hypothetically any of the data can be exposed publicly.
+
+1. After you share your dashboard, which uses part of the data, someone may create a new dashboard from your dataset - they may or may not notice that your dataset is public - which publicly exposes other parts of your dataset that you didn't expect.
+
+2. Other Superset users that have higher permissions, like running SQL queries against their own datasets, may also be able to use those higher permissions on your public dataset.
+
+For this reason, it's strongly recommended to take these steps unless all data in your dataset can TRULY be made public:
```

Comment:
For this reason, it's strongly recommended to take these steps unless all data in your dataset can be made completely public: